pkg/egressgateway/manager_privileged_test.go

```diff
@@ -56,6 +57,8 @@ const (
 
 	egressIP1   = "192.168.101.1"
 	egressCIDR1 = "192.168.101.1/24"
+	egressIP2   = "192.168.102.1"
+	egressCIDR2 = "192.168.102.1/24"
```

Comment:
Not that it matters, but maybe `192.168.102.0/24` looks more canonical?

Reply:
I would agree, but right now that change causes the tests to fail :thinking:. Needs a closer eye why that is.

Reply:
Ok, I see. I think code parses this CIDR as an IP address in ```func parseIPRule(sourceIP, destCIDR, egressIP string, ifaceIndex int)``` (it parses it as a CIDR, but then only uses the IP address part), didn't look too precise